src/explore-education-statistics-admin/src/services/publicationService.ts

```diff
@@ -135,9 +141,13 @@ const publicationService = {
     return client.get<MyPublication>(`/me/publication/${publicationId}`);
   },
 
-  getReleases(publicationId: string): Promise<ReleaseSummary[]> {
-    return client.get<ReleaseSummary[]>(
+  getReleases(
```

Comment:
Would consider renaming this `listReleases`

Reply:
Done - also renamed `GetReleaseParams` to `ListReleaseParams`.